puppeteer-tests/README.md

```diff
@@ -3,10 +3,9 @@
 1. npm install
 2. Create a project on localhost and copy its URL.
 3. run the performance test like this in the terminal: `EDITOR_URL=<project-url-you-just-copied> npm run start`
-4. can provide an env var HEADLESS=false to see what the editor is doing: `HEADLESS=false EDITOR_URL=<project-url-you-just-copied> npm run start`
-5. It's possible to provide a browser path if the bundled one is no use: `BROWSER='google-chrome' EDITOR_URL=<project-url-you-just-copied> npm run start`
-6. you will see console output with the frame numbers.
-7. That's all you need for debugging existing tests!
+4. It's possible to provide a browser path if the bundled one is no use: `BROWSER='google-chrome' EDITOR_URL=<project-url-you-just-copied> npm run start`
```

Comment:
Minor: all of these should be using `pnpm` rather than `npm`. I have no issue making that change afterwards since it's completely unrelated to this PR